test/python/tests/cli_controller/test_cli_controller.py

```diff
@@ -87,15 +87,20 @@ async def test_controller_list(mayastor_mod, create_replicas, create_nexus):
 @pytest.mark.asyncio
 async def test_controller_stats(mayastor_mod, create_replicas, create_nexus):
     nexus = mayastor_mod.get("ms3")
+    print("engine started")
     mscli = get_msclient().with_json_output().with_url(nexus.ip_address())
-
     # Should see exactly 2 controllers on the nexus instance.
     assure_controllers(mscli, create_replicas)
-
+    list_controller = mscli("controller", "list")
+    print(list_controller)
```

Comment:
You can remove these print statements used for debugging.

Reply:
Done